infra/growth/src/apollo/adapter.js

```diff
@@ -75,6 +75,7 @@ class ApolloAdapter {
     if (!data.visible) {
       return null
     }
+    const ruleHasNoUserData = data.status === null
```

Comment:
How about checking on data.ethAddress being undefined rather than on data.status ?

Reply:
Also how about renaming ruleHasNoUserData to omitUserData ?

Reply:
yup both good points thanks